modules/core/server/services/push.server.service.js

```diff
@@ -7,6 +7,26 @@ var _ = require('lodash'),
     url = (config.https ? 'https' : 'http') + '://' + config.domain,
     analyticsHandler = require(path.resolve('./modules/core/server/controllers/analytics.server.controller'));
 
+exports.notifyPushDeviceAdded = function(user, platform, callback) {
+
+  if (_.get(user, 'pushRegistration', []).length === 0) return callback();
+
+  var editAccountUrl = url + '/profile/edit/account';
+
+  var notification = {
+    title: 'Trustroots',
+    body: 'You just enabled Trustroots ' + platform + ' push notifications. Yay!',
+    click_action: analyticsHandler.appendUTMParams(editAccountUrl, {
+      source: 'push-notification',
+      medium: 'fcm',
+      campaign: 'device-added',
+      content: 'reply-to'
```

Comment:
`medium` could probably be just `push-notification`, I'm not sure if we gonna differentiate different platforms in the future but doesn't matter for now.

Since there is just one place (notification click) pointing to the same url, `content` can probably be left empty. It's used to make difference e.g. between different parts of email linking to the same URL (e.g. "footer" and "header"). ...I think.

They're explained in [this doc](https://support.google.com/analytics/answer/1033863#parameters) and at [query builder](https://ga-dev-tools.appspot.com/campaign-url-builder/)